version/news/NEWS-2024.12.0-kousa-dogwood.md

```diff
@@ -43,6 +43,7 @@
 - Added a countdown on the homepage for VS Code and Positron sessions to reflect the `session-timeout-kill-hours` setting and indicate when a session may be terminated if the user doesn't interact with it (rstudio-pro#6743)
 - The most recently opened folder is saved for Positron sessions (rstudio-pro#6874)
 - Added preview support for setting an active timeout, which logs out users after a period of time, regardless of activity. (rstudio-pro#5815)
+- Introduced a new session init container that copies session components to the session container, allowing updates to components independent of the session image. The session init container can be enabled in Kubernetes sessions by setting `launcher-sessions-init-container-enabled=1` in `rserver.conf`. Additionally, the init container uses a new multi-platform Linux session components package to support session images across different Linux distributions. (rstudio-pro#6821)
```

Comment:
Do we format init?

```suggestion
- Introduced a new session `init` container that copies session components to the session container, allowing updates to components independent of the session image. The session `init` container can be enabled in Kubernetes sessions by setting `launcher-sessions-init-container-enabled=1` in `rserver.conf`. Additionally, the `init` container uses a new multi-platform Linux session components package to support session images across different Linux distributions. (rstudio-pro#6821)
```

Reply:
I am not sure if I cover this in the style guide - I probably should. Let me know if this should be formatted and I can make an entry.

Reply:
FWIW, "init" is not a short-hand here, it's the full name of a feature in Kubernetes: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/

Reply:
Ah - >  init containers in the Pod  - roger that. Ignore my suggestions for this change!